modules/machineconfig-modify-registry.adoc

```diff
@@ -25,7 +25,7 @@ See the `containers-registries.conf` man page for the format for the `registries
 
 .Procedure
 
-. Create a YAML file (`myregistry.yaml`) to hold the contents of the `/etc/containers/registries.d/99-worker-unqualified-search-registries.conf` file, including the encoded base64 contents for that file. For example:
+. Create a YAML file (`myregistry.yaml`) to hold the contents of the `/etc/containers/registries.conf.d/99-worker-unqualified-search-registries.conf` file, including the encoded base64 contents for that file. For example:
```

Comment:
And here.

Reply:
`registries.conf.d` is actually what we need here as it is talking about adding a drop-in file to  modify `unqualified-search-registries`.

Reply:
@ahardin-rh looks like this is a bit outdated. In 4.7, we added a new option to the cluster wide `Image` CR - `containerRuntimeSearchRegistries`. Users can use that set their configured list of unqualified-search-registries and the controller rolls out the changes to the appropriate nodes. However, we heavily advice against using unqualified-search-registries and it was documented by https://github.com/openshift/openshift-docs/pull/28152/files.

I think we should not document how someone can do this with a MC, we want users to use our CRDs for making such changes to the node. Signatures are not supported yet, so using an MC for that makes sense, but that is something we are looking to combine into the `Image` CRD in the future.

Reply:
@umohnani8 Thank you! Should I create a new PR to remove this procedure from the 4.7+ doc set then? 

FYI @mburke5678

Reply:
Yes please, it should be removed from the 4.7 and later docs.

Reply:
@umohnani8 Thank you! I will get this work merged and then remove the procedure from 4.7 and 4.8 in a follow-up PR. Thanks!